### ISSUE TITLE
Update deps for httplib2 to 0.7.5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#194]
```
See issue #193 for the error that can occur if 0.7.5 isn't used.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 11 Sep 2012 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 
http://code.google.com/p/google-api-python-client/source/detail?r=9c82c71824fc7b
17035e8ff952734473043925b0 and is available in the 1.0 release.
```

Original comment by `jcgregorio@google.com` on 12 Sep 2012 at 6:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 12 Sep 2012 at 6:13
- Changed state: **Fixed**
